pootle/apps/pootle_data/utils.py

```diff
@@ -274,9 +274,11 @@ def aggregate_max_fields(self):
             for f
             in self.max_fields)
 
-    @property
+    @persistent_property
```

Comment:
I see that revision updater in store_data save handler can update "checks" however I couldn't find where it is used.
IIUC persistent_property for checks should use "checks" key in rev_cache_key though I could misunderstand.